scripts/templates/helper.py

```diff
@@ -695,6 +695,16 @@ def make_param_lines(namespace, tags, obj, py=False, decl=False, meta=None, form
         name = _get_param_name(namespace, tags, item)
         if py:
             tname = get_ctype_name(namespace, tags, item)
+            # Handle fptr_typedef
+            # On Python side, passing a function pointer to a CFUNCTYPE is a bit awkward
+            # So solve this, if we encounter a function pointer type, we relpace it with
+            # c_void_p - a generic void pointer
+            if "fptr_typedef" in meta:
+                fptr_types = list(meta['fptr_typedef'].keys())
```

Comment:
Couldn't we construct this list outside the for loop this is inside of?

Reply:
done